guides/manual-installation.md

```diff
@@ -615,7 +607,7 @@ server {
     ssl_session_cache shared:SSL:10m;
 
     # Wanda rule
-    location ~ ^/(api/checks|api/v1/checks|api/v2/checks)/  {
+    location ~ ^/(api/checks|api/v1/checks|api/v2/checks|api/v3/checks)/  {
```

Comment:
Hey @nelsonkopliku and @dottorblaster, we currently have redirect rules for this list of endpoints. What about suma integration? 
Do we need more rules?

Reply:
hey @EMaksy nope. Nothing suma related need at this level. It is all within trento APIs